charts/index.html

```diff
@@ -0,0 +1,16 @@
+<html>
+
+<head>
+    <title>Secrets Store CSI Driver Helm Chart Repository</title>
+</head>
+
+<body>
+    <h1>Secrets Store CSI Driver Helm Chart Repository</h1>
+
+    <hr>
+    <pre>
+Checkout the <a href="https://secrets-store-csi-driver.sigs.k8s.io/getting-started/installation.html#installation">installation guide</a> to install the Secrets Store CSI Driver and providers.
+    </pre>
+</body>
+
+</html>
```

Comment:
nit: add newline

Reply:
I tried adding it but it shows up as an extra new line for html which is weird!